### ISSUE TITLE
Support for pug

### ISSUE THREAD

**Issue by Almoullim** [tailwindlabs/tailwindcss-intellisense#233]
I know this is supported:
```
div.bg-red-500
```
hover, this is not:
```
div(class="bg-red-500")
```

**Comment by Almoullim** [comment]
Looks like it works if you add a space after the word class. which is annoying tbh.

for example:

this wont work
```
div(class="bg-red-500")
```
but this will:
```
div( class="bg-red-500")
```

**Comment by YilianSource** [comment]
Seems like this has been an issue for quite a while.

The workaround I used was to include a custom regex in `tailwindCSS.experimental.classRegex`:

```json
"tailwindCSS.experimental.classRegex": [
    "class=[\"']([^\"']*)[\"']"
]
```

This matches `class` attributes with either `'` or `"` quotes. Hope this helps!

**Comment by bradlc** [comment]
This used to work but there was a regression at some point. This has been resolved in `v0.5.7`, so `div(class="_")` should work out-of-the-box now 👍
